tap_jira/__init__.py

```diff
@@ -65,6 +65,10 @@ def generate_metadata(stream, schema):
     mdata = metadata.new()
     mdata = metadata.write(mdata, (), 'table-key-properties', stream.pk_fields)
```

Comment:
Can't we need to change PK before this? Else it will not change the PK inside metadata!!

Reply:
Updated. Actually, a later change in the value of `stream.pk_fields` is also reflected in metadata. That's why the order does not matter.